### ISSUE TITLE
Incorrect generated code for an overridden action with arguments

### ISSUE THREAD

**Issue by migig** [T4MVC/T4MVC#32]
Suppose there are two projects, "parent" and "child". Both have controllers, both use T4MVC.

Ideally you'd use abstract controllers/actions, but T4MVC doesn't work well that way. So you mark them as `virtual` and `override` instead. That works. But, not when there are arguments.

**This works:**

``` c#
namespace ParentProject {
    public partial class FooController : Controller {
        public virtual ActionResult DoStuff() {  // does nothing, but can call it in parent project
          throw new NotImplementedException();
        }
    }
}
namespace ChildProject {
    public partial class FooController : ParentProject.FooController {
        public override ActionResult DoStuff() {  // does the actual work 
            // do stuff
        }
    }
}
```

**This doesn't:**

``` c#
namespace ParentProject {
    public partial class FooController : Controller {
       public virtual ActionResult DoStuff(int id) {     // argument is the problem
          throw new NotImplementedException();
        }
    }
}
namespace ChildProject {
    public partial class FooController : ParentProject.FooController {
        public override ActionResult DoStuff(int id) {     // argument is the problem
          // do stuff
        }
    }
}
```

So when there is an argument, the generated code is buggy. The error on compilation is:
`Warning    'ChildProject.FooController.DoStuff()' hides inherited member 'ParentProject.FooController.DoStuff()'. To make the current member override that implementation, add the override keyword. Otherwise add the new keyword.`

**Comment by migig** [comment]
Changing the tt from this:

```
#pragma warning disable 1591, 3008, 3009, 0108
// .....
#pragma warning restore 1591, 3008, 3009, 0108
```

to this:

```
#pragma warning disable 1591, 3008, 3009, 0108, 0114
// .....
#pragma warning restore 1591, 3008, 3009, 0108, 0114
```

Works.

I wouldn't do this for any other code, as you might mask legit problems. But in T4MVC this only happens for overridden actions and so forth, and since T4MVC doesn't handle such edge cases properly, it's better to just hide the warning.

Hope to see this in the official template!

UPDATE: [I added a PR](https://github.com/T4MVC/T4MVC/pull/33).

**Comment by davidebbo** [comment]
Fix is in 3.15.4.

**Comment by migig** [comment]
:+1: :beer:
